src/commands/base-command.ts

```diff
@@ -192,17 +192,16 @@ export default class BaseCommand extends Command {
       .addOption(new Option('--cwd <cwd>').hideHelp(true))
       .addOption(new Option('-o, --offline').hideHelp(true))
       .addOption(new Option('--auth <token>', 'Netlify auth token').hideHelp(true))
+      // TODO(serhalp) Remove in next major release. This was intended to be deprecated in 2023 but a typo
+      // delayed it to Feb 2025.
```

Comment:
I could also see an argument for just renaming this one:
- this particular one was hidden from help to begin with, so you could argue it was undocumented
- we're cutting a major here anyway